etl/processor/processor.go

```diff
@@ -141,8 +130,8 @@ func (p *Processor) process(ctx context.Context, jd *types.JetDrop) {
 	}
 
 	mjd := models.JetDrop{
-		JetID:          ms.Start.JetDropPrefix,
-		PulseNumber:    pd.PulseNo,
+		JetID:          ms.Start.JetDropPrefix, // FIXME
+		PulseNumber:    pd.PulseNo,             //FIXME
```

Comment:
So how about fix this?

Reply:
what do you suggest ?

Reply:
But i think, everything is fine now. We got pulse from pulse.